### PR TITLE
chore: 0.2.9 release

### PR DESCRIPTION
**Pull request by HuiSF**
*Issue #, if available:*

*Description of changes:*

0.2.9 release integrating amplify-android (1.28.3-rc) with custom domain support of AppSync.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: packages/amplify_api/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 0.2.9 (2021-11-17)
+
+- chore: upgrade amplify-ios to 1.28.3-rc
```
Comment (dnys1) [inline packages/amplify_api/CHANGELOG.md:3]: amplify android yeah?

**Review by dnys1 — APPROVED** [review]
Thanks!

File: packages/amplify_api/CHANGELOG.md (hunk 1 above)
Comment (HuiSF) [reply packages/amplify_api/CHANGELOG.md]: Yup my bad. Updated.

**Review by ragingsquirrel3 — APPROVED** [review]
thanks! :shipit:

**Pull request merged**